package.json

```diff
@@ -8,10 +8,30 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "seed": "node ./server/models/seed.js",
+    "start": "nodemon server/index.js",
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js"
   },
   "dependencies": {
+    "@material-ui/core": "^4.2.0",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.6.4",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "jest": "^24.8.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "@babel/core": "^7.5.0",
```

Comment:
You don't seem to be using Babel or webpack as devDependencies. I would recommend moving them to `dependencies` instead.